### PR TITLE
Chg tr unit to sugg enum

### PR DESCRIPTION
**Pull request by aschroed**
Changed treatment_agent.units enum to suggested_enum
Added pfu to units.

### CODE REVIEW

File: src/encoded/schemas/treatment_agent.json [hunk 1]
```diff
@@ -93,7 +93,8 @@
                 "percent",
                 "units",
                 "U/mL",
-                "MOI"
+                "MOI",
+                "pfu"
```
Comment (andreacosolo) [inline src/encoded/schemas/treatment_agent.json:97]: I'd suggest to write PFU in capital letters, to be consistent with MOI (or both small?). Unless this is the way they are most commonly used (I'm not too familiar with these units).

**Review by andreacosolo — APPROVED** [review]
Just a non-blocking comment on notation.

File: src/encoded/schemas/treatment_agent.json [hunk 2]
```diff
@@ -81,7 +81,7 @@
             "title": "Concentration units",
             "type": "string",
             "lookup": 40,
-            "enum": [
+            "suggest_enum": [
```
Comment (sbreiff) [inline src/encoded/schemas/treatment_agent.json:84]: doesn't this need to be "suggested_enum"?

File: src/encoded/schemas/treatment_agent.json (hunk 1 above)
Comment (sbreiff) [reply src/encoded/schemas/treatment_agent.json:97]: both capital, IMO.

File: src/encoded/schemas/treatment_agent.json (hunk 2 above)
Comment (aschroed) [reply src/encoded/schemas/treatment_agent.json]: Of course it does - this is exactly why we do reviews even for small changes like this.  Doh! Thanks. Fixed now I hope.

**Review by sbreiff — CHANGES_REQUESTED** [review]
change suggest_ to suggested_. [EDIT: you've just done this so I approve.] I also agree with Andrea about the capitals.

One other question though - is it total PFU? Or PFU/ml? These are concentration units and "PFU" isn't a concentration.

**Review by sbreiff — APPROVED** [review]

**Pull request merged**